src/pages/Admin/Cities/EditCity/EditCity.jsx

```diff
@@ -24,18 +24,37 @@ const EditCity = () => {
       .getCityById(id)
       .then((city) => {
         setCityName(city.name)
-        setPriceForHour(city.priceForHour)
+        setPriceForHour(formatValueToDecimal(city.priceForHour))
       })
       .then(() => setIsLoadnig(false))
   }, [id])
 
-  const onBlurCity = () => {
-    setCityError('')
-    setError('')
+  const changePriceForHour = (event) => {
+    let priceForHour = event.target.value
+    priceForHour = priceForHour.replace(/,/g, '.')
+    priceForHour = priceForHour.replace(/[^\d.]/g, '')
+
+    if (parseFloat(priceForHour) < 0) {
+      event.preventDefault()
+      return
+    }
+    const dotIndex = priceForHour.indexOf('.')
+    if (dotIndex !== -1 && priceForHour.indexOf('.', dotIndex + 1) !== -1) {
+      event.preventDefault()
+      return
+    }
+    const decimalIndex = priceForHour.indexOf('.')
+    if (decimalIndex !== -1 && priceForHour.substring(decimalIndex + 1).length > 2) {
+      event.preventDefault()
+      return
+    }
+    setPriceForHour(priceForHour)
   }
-  const onBlurPriceForHour = () => {
-    setPriceForHourError('')
-    setError('')
+  const formatValueToDecimal = (value) => {
```

Comment:
Create own file like helpers.js and put formatValueToDecimal and formatValueToInteger to this file, and use by import in each place where you need